jobbergate-api/jobbergate_api/tests/apps/job_submissions/test_routers.py

```diff
@@ -208,6 +238,10 @@ async def test_create_job_submission__with_execution_directory(
     assert job_submission.created_at in window
     assert job_submission.updated_at in window
 
+    assert job_submission.execution_parameters is not None
+    assert job_submission.execution_parameters.name == "job-submission-name"
+    assert job_submission.execution_parameters.comment == "I am a comment"
```

Comment:
again

Reply:
in this one the code actually call `database.fetch_all`, so it is just a case of improving a little bit to check if the data in the database is exactly what we want to.